2024-04-08 11:00:28,882 INFO    MainThread:25300 [wandb_setup.py:_flush():76] Current SDK version is 0.16.6
2024-04-08 11:00:28,883 INFO    MainThread:25300 [wandb_setup.py:_flush():76] Configure stats pid to 25300
2024-04-08 11:00:28,883 INFO    MainThread:25300 [wandb_setup.py:_flush():76] Loading settings from C:\Users\HP\.config\wandb\settings
2024-04-08 11:00:28,883 INFO    MainThread:25300 [wandb_setup.py:_flush():76] Loading settings from e:\Data Analysis\LoanPredictionSystemML\wandb\settings
2024-04-08 11:00:28,883 INFO    MainThread:25300 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-04-08 11:00:28,883 INFO    MainThread:25300 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-04-08 11:00:28,883 INFO    MainThread:25300 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-04-08 11:00:28,883 INFO    MainThread:25300 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-04-08 11:00:28,883 INFO    MainThread:25300 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-04-08 11:00:28,883 INFO    MainThread:25300 [wandb_init.py:_log_setup():521] Logging user logs to e:\Data Analysis\LoanPredictionSystemML\wandb\run-20240408_110028-kgnow5rc\logs\debug.log
2024-04-08 11:00:28,883 INFO    MainThread:25300 [wandb_init.py:_log_setup():522] Logging internal logs to e:\Data Analysis\LoanPredictionSystemML\wandb\run-20240408_110028-kgnow5rc\logs\debug-internal.log
2024-04-08 11:00:28,883 INFO    MainThread:25300 [wandb_init.py:_jupyter_setup():467] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000002D9ADD13D50>
2024-04-08 11:00:28,883 INFO    MainThread:25300 [wandb_init.py:init():561] calling init triggers
2024-04-08 11:00:28,883 INFO    MainThread:25300 [wandb_init.py:init():568] wandb.init called with sweep_config: {}
config: {}
2024-04-08 11:00:28,883 INFO    MainThread:25300 [wandb_init.py:init():586] re-initializing run, found existing run on stack: c3edr97h
2024-04-08 11:00:28,885 INFO    MainThread:25300 [wandb_run.py:_finish():2064] finishing run ericmaniraguha/loan_prediction_project/c3edr97h
2024-04-08 11:00:28,885 INFO    MainThread:25300 [wandb_run.py:_atexit_cleanup():2304] got exitcode: 0
2024-04-08 11:00:28,886 INFO    MainThread:25300 [wandb_run.py:_restore():2287] restore
2024-04-08 11:00:28,886 INFO    MainThread:25300 [wandb_run.py:_restore():2293] restore done
2024-04-08 11:00:34,960 INFO    MainThread:25300 [wandb_run.py:_footer_history_summary_info():3936] rendering history
2024-04-08 11:00:34,960 INFO    MainThread:25300 [wandb_run.py:_footer_history_summary_info():3968] rendering summary
2024-04-08 11:00:34,961 INFO    MainThread:25300 [wandb_run.py:_footer_sync_info():3895] logging synced files
2024-04-08 11:00:35,133 INFO    MainThread:25300 [wandb_init.py:init():611] starting backend
2024-04-08 11:00:35,133 INFO    MainThread:25300 [wandb_init.py:init():615] setting up manager
2024-04-08 11:00:35,144 INFO    MainThread:25300 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-04-08 11:00:35,144 INFO    MainThread:25300 [wandb_init.py:init():623] backend started and connected
2024-04-08 11:00:35,144 INFO    MainThread:25300 [wandb_run.py:_label_probe_notebook():1299] probe notebook
2024-04-08 11:00:35,144 INFO    MainThread:25300 [wandb_run.py:_label_probe_notebook():1309] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-04-08 11:00:35,144 INFO    MainThread:25300 [wandb_init.py:init():715] updated telemetry
2024-04-08 11:00:35,179 INFO    MainThread:25300 [wandb_init.py:init():748] communicating run to backend with 90.0 second timeout
2024-04-08 11:00:36,892 INFO    MainThread:25300 [wandb_run.py:_on_init():2357] communicating current version
2024-04-08 11:00:38,044 INFO    MainThread:25300 [wandb_run.py:_on_init():2366] got version response 
2024-04-08 11:00:38,044 INFO    MainThread:25300 [wandb_init.py:init():799] starting run threads in backend
2024-04-08 11:00:39,291 INFO    MainThread:25300 [wandb_run.py:_console_start():2335] atexit reg
2024-04-08 11:00:39,291 INFO    MainThread:25300 [wandb_run.py:_redirect():2190] redirect: wrap_raw
2024-04-08 11:00:39,291 INFO    MainThread:25300 [wandb_run.py:_redirect():2255] Wrapping output streams.
2024-04-08 11:00:39,291 INFO    MainThread:25300 [wandb_run.py:_redirect():2280] Redirects installed.
2024-04-08 11:00:39,291 INFO    MainThread:25300 [wandb_init.py:init():842] run started, returning control to user process
2024-04-08 11:00:39,315 INFO    MainThread:25300 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-08 11:00:39,315 INFO    MainThread:25300 [wandb_init.py:_pause_backend():432] pausing backend
2024-04-08 11:01:00,860 INFO    MainThread:25300 [wandb_init.py:_resume_backend():437] resuming backend
2024-04-08 11:01:00,876 INFO    MainThread:25300 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-08 11:01:00,876 INFO    MainThread:25300 [wandb_init.py:_pause_backend():432] pausing backend
2024-04-08 11:01:07,714 INFO    MainThread:25300 [wandb_init.py:_resume_backend():437] resuming backend
2024-04-08 11:01:08,119 INFO    MainThread:25300 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-08 11:01:08,119 INFO    MainThread:25300 [wandb_init.py:_pause_backend():432] pausing backend
2024-04-08 11:01:08,959 INFO    MainThread:25300 [wandb_init.py:_resume_backend():437] resuming backend
2024-04-08 11:01:08,991 INFO    MainThread:25300 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-08 11:01:08,991 INFO    MainThread:25300 [wandb_init.py:_pause_backend():432] pausing backend
2024-04-08 11:01:10,070 INFO    MainThread:25300 [wandb_init.py:_resume_backend():437] resuming backend
2024-04-08 11:01:10,078 INFO    MainThread:25300 [wandb_setup.py:_flush():76] Current SDK version is 0.16.6
2024-04-08 11:01:10,079 INFO    MainThread:25300 [wandb_setup.py:_flush():76] Configure stats pid to 25300
2024-04-08 11:01:10,079 INFO    MainThread:25300 [wandb_setup.py:_flush():76] Loading settings from C:\Users\HP\.config\wandb\settings
2024-04-08 11:01:10,079 INFO    MainThread:25300 [wandb_setup.py:_flush():76] Loading settings from e:\Data Analysis\LoanPredictionSystemML\wandb\settings
2024-04-08 11:01:10,079 INFO    MainThread:25300 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-04-08 11:01:10,079 INFO    MainThread:25300 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-04-08 11:01:10,080 INFO    MainThread:25300 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-04-08 11:01:10,080 INFO    MainThread:25300 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-04-08 11:01:10,080 INFO    MainThread:25300 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-04-08 11:01:10,080 INFO    MainThread:25300 [wandb_init.py:_log_setup():521] Logging user logs to e:\Data Analysis\LoanPredictionSystemML\wandb\run-20240408_110110-x2oi5mr7\logs\debug.log
2024-04-08 11:01:10,080 INFO    MainThread:25300 [wandb_init.py:_log_setup():522] Logging internal logs to e:\Data Analysis\LoanPredictionSystemML\wandb\run-20240408_110110-x2oi5mr7\logs\debug-internal.log
2024-04-08 11:01:10,080 INFO    MainThread:25300 [wandb_init.py:init():561] calling init triggers
2024-04-08 11:01:10,080 INFO    MainThread:25300 [wandb_init.py:init():568] wandb.init called with sweep_config: {}
config: {}
2024-04-08 11:01:10,080 INFO    MainThread:25300 [wandb_init.py:init():586] re-initializing run, found existing run on stack: kgnow5rc
2024-04-08 11:01:10,082 INFO    MainThread:25300 [wandb_run.py:_finish():2064] finishing run ericmaniraguha/loan_prediction_project/kgnow5rc
2024-04-08 11:01:10,083 ERROR   MainThread:25300 [jupyter.py:save_history():437] Run pip install nbformat to save notebook history
2024-04-08 11:01:10,083 INFO    MainThread:25300 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-08 11:01:10,083 INFO    MainThread:25300 [wandb_init.py:_jupyter_teardown():449] cleaning up jupyter logic
2024-04-08 11:01:10,083 INFO    MainThread:25300 [wandb_run.py:_atexit_cleanup():2304] got exitcode: 0
2024-04-08 11:01:10,084 INFO    MainThread:25300 [wandb_run.py:_restore():2287] restore
2024-04-08 11:01:10,084 INFO    MainThread:25300 [wandb_run.py:_restore():2293] restore done
2024-04-08 11:01:14,608 INFO    MainThread:25300 [wandb_run.py:_footer_history_summary_info():3936] rendering history
2024-04-08 11:01:14,608 INFO    MainThread:25300 [wandb_run.py:_footer_history_summary_info():3968] rendering summary
2024-04-08 11:01:14,608 INFO    MainThread:25300 [wandb_run.py:_footer_sync_info():3895] logging synced files
